lib/Quantization/Quantization.cpp

```diff
@@ -333,6 +338,28 @@ class FunctionQuantizer : public FunctionConverter {
 
       break;
     }
+
+      // For Select, need to set LHS and RHS inputs with same quantization
```

Comment:
Should the comment block here be aligned with the `case` below? If so, it doesn't appear to be.

Reply:
This is how `clang-format` does it, unfortunately. I prefer it to be aligned with the case. I could always move the comment into the case so it looks more natural.